docs/modules/ROOT/pages/advanced-how-tos/testing-releasing-single-component.adoc

```diff
@@ -0,0 +1,200 @@
+= Testing and Releasing a Single Component
+
+By default, Konflux treats applications as a set of dependent components. A example would frontend and backend
+components. Typically, a team would want to build these components, while testing and releasing them together
+to ensure there are compatible.
+
+However, applications can also be viewed simply as a group of non-dependent components.
+
+Consider this:
+
+* You have an application that contains 5 components.
+* You need to release a single component to fix a bug that a customer is experiencing.
+* While attempting to release, another component has a new violation therefore causing the Enterprise Contract
+policy validation to fail.
+** This violation may be due to a rule that used to be a warning but has expired and is now a violation.
+
+Wouldn't it be nice to be able to ignore that failing component and simply release the component you are targeting?
+
+Enter **Single Component Mode**!
+
+== Enabling Single Component Mode
+
+Single component mode works by analyzing the Snapshot that is being used for Integration Tests and Releasing.
+A Snapshot created due to a single component being built is the only Snapshot type that may be used for this
+mode.
+
+=== Enabling Single Component Mode for Enterprise Contract Integration Test Scenarios
+
+Single Component Mode can be enabled either by making a change to the Integration Test Scenario custom resource
+or by using the HAC UI.
+
+* Update the custom resource by adding the parameter *SINGLE_COMPONENT* with a value of *true*
+
+[source,yaml]
+.integration-test-scenario.yaml
+----
+apiVersion: appstudio.redhat.com/v1beta2
+kind: IntegrationTestScenario
+metadata:
+  name: grouping-enterprise-contract-single-component
+  namespace: shebert-tenant
+spec:
+  application: grouping
+  contexts:
+    - description: Application testing
+      name: application
+  params:
```

Comment:
Is this the only part that is new? If so, we can just use a snippet like we do for defining build customizations so that they don’t need to figure out what information they don’t need to include.

Reply:
fixed snippet